com.unity.build-report-inspector/README.md

```diff
@@ -50,6 +47,34 @@ For platforms which support engine code stripping, a list of all engine modules
 
 ### Scenes using Assets
 [Available from Unity 2020.1.0a6]  
-When BuildOption.DetailedBuildReport is passed to [BuildPipeline.BuildPlayer](https://docs.unity3d.com/ScriptReference/BuildPipeline.BuildPlayer.html), a list describing which scenes are using each asset of the build, is provided in the BuildReport.
+When BuildOptions.DetailedBuildReport is passed to [BuildPipeline.BuildPlayer](https://docs.unity3d.com/ScriptReference/BuildPipeline.BuildPlayer.html), a list describing which scenes are using each asset of the build, is provided in the BuildReport.
 
 <img src="Documentation~/images/ScenesUsingAssets.png" width="400">
+
```

Comment:
I think you need to update file com.unity.build-report-inspector/Documentation~/com.unity.build-report-inspector.md as well, it is the source for online package documentation page https://docs.unity3d.com/Packages/com.unity.build-report-inspector@0.1/manual/index.html

Reply:
Mirrored the same info to com.unity.build-report-inspector.md, thanks